docs/docs/deploy-gatsby.md

```diff
@@ -80,12 +80,20 @@ The repository for these sites requires a special name. See
 https://help.github.com/articles/user-organization-and-project-pages/ for
 documentation on naming your site's repository.
 
+If you wish to use custom domain with your `user.github.io` repo, you will need
+a `CNAME` file at the root directory level with the your custom domain url inside, 
+like so:
+
+```
+example.com
+```
+
 Like with project sites, add `gh-pages` as a `devDependency` and add a `deploy`
 script to your site's `package.json` file:
 
 ```
 "scripts": {
-  "deploy": "gatsby build && gh-pages -d public --branch master",
+  "deploy": "gatsby build && cp CNAME public/ && gh-pages -d public --branch master",
```

Comment:
We recommend using the `static` folder for this — https://www.gatsbyjs.org/docs/adding-images-fonts-files/#using-the-static-folder

Reply:
Hi Kyle, that would mean the CNAME file would be under the `static/` folder, for custom domain to work, `CNAME` has to be under the root folder

Reply:
@bodaz Files you place in you project `static` directory are copied to root output directory `/public` (not `/public/static`) so it's perfect place for it.

Reply:
@pieh 👍  thanks, I misread the docs 😞